content/news/2020-10-04-wallpaper-contest.md

```diff
@@ -39,7 +39,13 @@ There is no strict deadline for the call, we would like you to take as much time
 
 ## Current Submissions
 
-- Julius Enriquez - [Linear Leaf (whole pack)](files/wallpaper-call/linear_leaf.zip) (added 2020/10/16)
+Cartoon Network Leaks (added 2020/10/17)
+
+<img src="/files/wallpaper-call/cartoon_network_leaks.png" height="270" width="480" />
+
+- Julius Enriquez - [Falling Leaf](whole pack)(files/wallpaper-call/falling_leaf.zip) (added 2020/10/17), [Linear Leaf (whole pack)](files/wallpaper-call/linear_leaf.zip) (added 2020/10/16)
```

Comment:
```suggestion
- Julius Enriquez - [Falling Leaf (whole pack)](/files/wallpaper-call/falling_leaf.zip) (added 2020/10/17), [Linear Leaf (whole pack)](/files/wallpaper-call/linear_leaf.zip) (added 2020/10/16)
```
Plus, I thinks it's better to have have each preview right beneath its link to the whole-pack-zip link.